app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -43,7 +45,9 @@ echo "the grub2_chainload template."
 echo
 echo "(*) grub2-efi-x64-2.02-122.el8 (upstream doesn't have the patches yet)"
 echo
-#connectefi scsi
+<% if host_param_true?('grub2-connectefi-scsi', true) -%>
+connectefi scsi
```

Comment:
Can we also parametrize this line? It can be either `scsi` or `pciroot`

Reply:
Also: since it could generate an error in the UI, maybe we should also `echo` a comment here that the error here is expected?